CONTRIBUTING.md

```diff
@@ -84,14 +84,12 @@ be sure to check out the awesome [article about forking](https://help.github.com
 on the GitHub Help website &mdash; it will get you started quickly.
 
 You should always write each batch of changes (feature, bugfix, etc.) in **its own topic branch**.
-Please do not commit to the `main` branch, or your unrelated changes will go into the same pull request.
+Please do not commit to the `main` branch of your fork else your unrelated changes will go into the same pull request.
 
 You should also follow the code style and whitespace conventions of the original codebase.
 In particular, use tabs for indentation and spaces for alignment.
 
-Before committing your changes, run `npm run lint` to catch any JS errors in the code and fix them.
-If you add any new files to the Leaflet source, make sure to also add them to `build/deps.js`
-so that the build system knows about them.
+Before committing your changes, run `npm run lint` to catch any JS errors in the code and fix them. The same command is automatically executed while committing. You can prevent it from execution with the git flag `--no-verify`: `git commit -m "WIP" --no-verify`.  
```

Comment:
Nit: let's keep each sentence on a separate line like in the rest of the doc. Makes maintenance easier.

```suggestion
Before committing your changes, run `npm run lint` to catch any JS errors in the code and fix them. 
The same command is automatically executed while committing. 
You can prevent it from execution with the git flag `--no-verify`: `git commit -m "WIP" --no-verify`.  
```